docs/quickstart.rst

```diff
@@ -13,7 +13,7 @@ create a minimal application, or use its scaffolding tool:
 
 ::
 
-    pcreate -s starter MyProject
+    $ pcreate -s starter MyProject
```

Comment:
I always wondered if we need to add these `$` signs everywhere. They just tend to make copy/paste harder in my opinion.